packages/components/src/form-token-field/test/index.tsx

```diff
@@ -205,6 +206,45 @@ describe( 'FormTokenField', () => {
 			] );
 		} );
 
+		it( 'should add a token with the input value with onBlur when `tokenizeOnBlur` prop is `true`', async () => {
+			const user = userEvent.setup();
+
+			const onChangeSpy = jest.fn();
+
+			const { rerender } = render(
+				<FormTokenFieldWithState onChange={ onChangeSpy } />
+			);
+
+			const input = screen.getByRole( 'combobox' );
+
+			// Add 'grapefruit' token by typing it and check blur of field does not tokenize it.
+			await user.type( input, 'grapefruit' );
+			act( () => {
+				input.blur();
+			} );
+			expect( onChangeSpy ).toHaveBeenCalledTimes( 0 );
+			expectTokensNotToBeInTheDocument( [ 'grapefruit' ] );
+
+			rerender(
+				<FormTokenFieldWithState
+					onChange={ onChangeSpy }
+					tokenizeOnBlur
+				/>
+			);
+			await user.clear( input );
+
+			// Add 'grapefruit' token by typing it and check blur of field tokenizes it.
+			await user.type( input, 'grapefruit' );
+
+			act( () => {
+				input.blur();
+			} );
+			expect( onChangeSpy ).toHaveBeenNthCalledWith( 1, [
+				'grapefruit',
+			] );
+			expectTokensToBeInTheDocument( [ 'grapefruit' ] );
+		} );
+
 		it( "should not add a token with the input's value when pressing the tab key", async () => {
```

Comment:
```suggestion
		it( "should not add a token with the input's value when tokenizeOnBlur is not set and pressing the tab key", async () => {
```

We should probably make this test description more accurate as well.

Reply:
That test is actually the existing one following the one I added, and it does test the tab key I think 😃

Reply:
It is the existing test, however, it should only not add the token when hitting tab when tokenizeOnBlur is not set, hence the clarification in the description. If you add `tokenizeOnBlue` to that test's component render, it will fail.

Am I missing something?

Reply:
> Am I missing something?

No but I was, I see what you mean now, fixed.